core/templates/pages/exploration-editor-page/services/parameter-metadata.service.ts

```diff
@@ -95,15 +95,11 @@ angular.module('oppia').factory('ParameterMetadataService', [
 
       // Finally, the rule feedback strings are evaluated.
       state.interaction.answerGroups.forEach(function(group) {
-        for (var k = 0; k < group.outcome.feedback.length; k++) {
-          ExpressionInterpolationService.getParamsFromString(
-            group.outcome.feedback[k]).forEach(
-            function(paramName) {
-              result.push(ParamMetadataObjectFactory.createWithGetAction(
-                paramName, PARAM_SOURCE_FEEDBACK, k));
-            }
-          );
-        }
+        ExpressionInterpolationService.getParamsFromString(
+          group.outcome.feedback.getHtml()).forEach(function(paramName, index) {
```

Comment:
Isn't group.outcome.feedback an array? (like in the original code) Why this change?

Reply:
According to `OutcomeObjectFactory.ts`, feedback is a `SubtitledHtml` object. `ExpressionInterpolationService.getParamsFromString` expects a string as a parameter, so that's why I changed it to `getHtml()`